src/coreclr/jit/hwintrinsic.cpp

```diff
@@ -832,6 +834,9 @@ static const HWIntrinsicIsaRange hwintrinsicIsaRangeArray[] = {
     { NI_Illegal, NI_Illegal },                                 // AVX512VBMI_X64
     { FIRST_NI_AVX10v1_X64, LAST_NI_AVX10v1_X64 },
     { NI_Illegal, NI_Illegal },                                 // AVX10v1_V512_X64
+    { NI_Illegal, NI_Illegal },                                 // APX_X64
```

Comment:
APX_X64 is unused.  It would be better to remove it from InstructionSetDesc.txt than to add the entry for it here.

Reply:
APX CPUID was enabled earlier today. Although it it unused, these are auto generated ISAs and they are added whenever we define a new ISA in ```InstructionSetDesc.txt```. I don't think removing ```APX_X64``` is an option here.

Reply:
It's added by this line, which shouldn't have been added since it's not needed.

https://github.com/dotnet/runtime/blob/30661f53e1a60d0d18d21488f4ab55e7e3621030/src/coreclr/tools/Common/JitInterface/ThunkGenerator/InstructionSetDesc.txt#L97

Removing that and re-generating will get rid of it everywhere.

Reply:
It was standard practice, because that's what the tool was setup for. However, the logic for the generator was recently updated and so it shouldn't be required anymore if there is no actual need for it.

Reply:
We want `AVX10v2_X64` because it has a managed API surface exposed. @saucecontrol was calling out `APX_X64` in particular, which is really unrelated to this PR and something to clean up independently

Reply:
I'm fine with either adding the `hwintrinsicIsaRangeArray` for `APX_X64` here and then it being removed in a follow up -or- just removing `InstructionSet_APX_X64` in this PR directly so that the range doesn't need to be added at all.

Will leave it up to you on what you think is easiest/best.

Reply:
I have removed ```APX_X64``` ISA.